src/store.js

```diff
@@ -0,0 +1,5 @@
+import { Provider } from 'react-redux';
+import { createStore } from "redux";
+
+
+export default store;
```

Comment:
Is `store` actually defined here?

Reply:
Ah, my bad. I removed the code I was playing with around and forgot to remove few things. This is one of those.